config/config-package.py

```diff
@@ -45,11 +45,11 @@ def handle_command_line_arguments():
         default=True,
         help='Prevent automatic committing of changes. Implies --no-push.')
```

Comment:
`--no-commit` says it implies `--no-push`, which no longer exists after this PR.

Reply:
Oh, right 🥽 I will fix it right away 😄